README.md

```diff
@@ -1,10 +1,22 @@
 The CHAPI protocol allows your digital wallet to receive Verifiable Credentails from an independent third-party issuer - or present Verifiable Credentials to an independent third-party verifier - in a way that establishes trust and preserves privacy.
 ## CHAPI is for open for _everyone_
-CHAPI is an open protocol designed to solve the "NASCAR Problem" - too often, users are presented with a fixed set of options for authentication with third-party sites.  The CHAPI protocol provides mediation between any CHAPI-enabled wallet and web application.  Just register your wallet with your browser, and off you go!
+CHAPI is an open protocol designed to solve the "NASCAR Problem" - too often, users are presented with a fixed set of options for authentication with third-party sites.  The CHAPI protocol provides mediation between any CHAPI-enabled web application and third party site.  Just register your web app with your browser, and off you go!
```

Comment:
```suggestion
CHAPI is an open protocol designed to solve the "NASCAR Problem" - too often, individuals are presented with a fixed set of options for authentication with third-party sites.  The CHAPI protocol provides mediation between any CHAPI-enabled web application and third party site.  Just register your web app with your browser, and off you go!
```

"There are only two industries which refer to their customers as users; illegal drugs and computers" -- Edward Tufte

There are a growing number of essays on why we should stop using the word "user" to describe the people we build systems for (two linked below):

https://uxdesign.cc/i-stopped-designing-for-users-and-began-designing-for-humans-instead-724e642d9dea
https://johannesippen.com/2019/humans-not-users/

I know it's nitpicky, and people will disagree about the use of the word "user" being lazy and/or derogatory in the computer industry... but, given that we're trying to pull people in, and a subset of those people (including me) are distracted by this language, let's try for something like this instead:

user -> individual / person
"their users" -> their visitors

specific type of user -> name of specific type, such as developer, teacher, architect, programmer, etc.

Reply:
+1